authorization/client_test.go

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2023 IndyKite
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package authorization_test
+
+import (
+	"context"
+
+	"github.com/golang/mock/gomock"
+
+	"github.com/indykite/indykite-sdk-go/authorization"
+	authorizationmock "github.com/indykite/indykite-sdk-go/test/authorization/v1beta1"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("IsAuthorized", func() {
+	var (
+		mockCtrl            *gomock.Controller
+		mockClient          *authorizationmock.MockAuthorizationAPIClient
+		authorizationClient *authorization.Client
+	)
+
+	BeforeEach(func() {
+		mockCtrl = gomock.NewController(GinkgoT())
+		mockClient = authorizationmock.NewMockAuthorizationAPIClient(mockCtrl)
+	})
+
+	Describe("Client", func() {
+		It("New", func() {
+			var err error
+			authorizationClient, err = authorization.NewTestClient(context.Background(), mockClient)
+			Ω(err).To(Succeed())
```

Comment:
Why sometimes you use `Ω` and sometimes `Expect`. Please align and use only one.